spec/requests/topics_controller_spec.rb

```diff
@@ -3743,7 +3743,7 @@ def extract_post_stream
 
     describe "when logged in" do
       before { sign_in(user) }
-      let!(:operation) { { type: "change_category", category_id: "1" } }
+      let!(:operation) { { type: "change_category", category_id: "1", silent: true } }
```

Comment:
There is a test  below

```
      it "delegates work to `TopicsBulkAction`" do
        topics_bulk_action = mock
        TopicsBulkAction
          .expects(:new)
          .with(user, topic_ids, operation, group: nil)
          .returns(topics_bulk_action)
        topics_bulk_action.expects(:perform!)

        put "/topics/bulk.json", params: { topic_ids: topic_ids, operation: operation }
      end
```

Reply:
It will definitely receives it, because the delegate test requires `operation` to appear in 
```ruby
TopicsBulkAction
          .expects(:new)
          .with(user, topic_ids, operation, group: nil)
```
 as is. I am very sure of this, because it took me an hour in the afternoon to realize that in the previous commit, `operation[:silent] = "true"` instead of `true` (string, not boolean), and this test point was always failed :cry: